tfx/utils/io_utils.py

```diff
@@ -140,9 +151,23 @@ def load_csv_column_names(csv_file: Text) -> List[Text]:
     return f.readline().strip().split(',')
 
 
-def all_files_pattern(file_pattern: Text) -> Text:
-  """Returns file pattern suitable for Beam to locate multiple files."""
-  return os.path.join(file_pattern, '*')
+def files_pattern(files_path: Text,
+                  pattern: Text = None) -> Text:
```

Comment:
Do you see a case where at the callsite we'd want to call this without supplying a file pattern? This function doesn't do anything if there is no file pattern provided, so I'd vote for making it required and removing the `None` default.